classes/classes/IMutationPerkType.as

```diff
@@ -71,7 +71,7 @@ public class IMutationPerkType extends PerkType
 
 			//Sets up the buff for the perk.
 			function setBuffs():void{
-				var stname:String = "mutation_" + mutations.id;
+				var stname:String = "perk_" + mutations.id;
```

Comment:
`"perk_"` prefix is checked in StatUtils.describeBuffs to merge all perk buffs into single entry